### ISSUE TITLE
Problem with Css reload

### ISSUE THREAD

**Issue by eocweb** [eddiemachado/bones#827]
Hi There this is The first time i'm using Bones.
I have a problem with Css/scss. When i edit One file scss (for example base.scss) adding new styles, i compile The style.scss and upload The new Style.css in The library folder. 
The file uploaded is ok but in my chrome, Safari or Firefox browser i cant see The change. 
I'm working online.
Thanks.

**Comment by Twintails** [comment]
The old file is probably cached. you'll need to clear cache.

**Comment by eocweb** [comment]
Yes first idea but cache Was deleted in every browser. For chrome i've tried cache killer too

**Comment by Twintails** [comment]
Your server may also be retaining a Cache, perhaps a plugin like WordFence or some such. If you are on a Hosted WP Solution, some Hosts optimize Apache Caching for you. 

That is to say, check the plugins first for a cache engine. Then discuss with your host. 

Optimally, you may want to DEV on an instance on your local Machine. This is so you don't push up breaking changes without a test environment to check them out. 

TeamTreehouse has some great training on Local Dev Environments.
